packages/squashfs.rb

```diff
@@ -2,46 +2,53 @@
 
 class Squashfs < Package
   description 'Squashfs is a compressed read-only filesystem for Linux.'
-  homepage 'http://squashfs.sourceforge.net/'
-  version '4.5'
+  homepage 'https://github.com/plougher/squashfs-tools'
+  version '4.6.1'
   license 'GPL-2'
   compatibility 'all'
-  source_url 'https://downloads.sourceforge.net/project/squashfs/squashfs/squashfs4.5/squashfs4.5.tar.gz'
-  source_sha256 'c493b29c3d152789d04fae5e6532499d96ce3f79256bc6df4f97b5170c88e979'
+  source_url 'https://github.com/plougher/squashfs-tools.git'
+  git_hashtag version
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/squashfs/4.5_armv7l/squashfs-4.5-chromeos-armv7l.tpxz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/squashfs/4.5_armv7l/squashfs-4.5-chromeos-armv7l.tpxz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/squashfs/4.5_i686/squashfs-4.5-chromeos-i686.tpxz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/squashfs/4.5_x86_64/squashfs-4.5-chromeos-x86_64.tpxz'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/squashfs/4.6.1_armv7l/squashfs-4.6.1-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/squashfs/4.6.1_armv7l/squashfs-4.6.1-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/squashfs/4.6.1_i686/squashfs-4.6.1-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/squashfs/4.6.1_x86_64/squashfs-4.6.1-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '630e363cf16b40b181ac3e09792ae9d7f73874ea4e902905824785c7c489546d',
-     armv7l: '630e363cf16b40b181ac3e09792ae9d7f73874ea4e902905824785c7c489546d',
-       i686: 'b72fc434de5a0c0f80edadff18efb2dfc13bc3223fe7a57b366621c1b34572f4',
-     x86_64: '6a3f77adbef01926ed731f7a69dab7607218dd13b31dda230556d5590dd7eea3'
+    aarch64: 'fa146db645ee4d4cf3b10f9a7e6425c182fef04505f289378a456e7aba7a373a',
+     armv7l: 'fa146db645ee4d4cf3b10f9a7e6425c182fef04505f289378a456e7aba7a373a',
+       i686: '51f3a8423f782ce0ee19ede8834deac84aa4161e321df87a397b2b92185395c8',
+     x86_64: '861359978b79ed8c8e6d8869b0a7b6c68ead651a627014f45bd9db65c5d58ec6'
   })
 
   depends_on 'compressdoc' => :build
+  depends_on 'gcc_lib' # R
+  depends_on 'glibc' # R
   depends_on 'help2man' => :build
-  depends_on 'lzo'
+  depends_on 'lz4' # R
+  depends_on 'lzo' # R
+  depends_on 'xzutils' # R
+  depends_on 'zlibpkg' # R
+  depends_on 'zstd' # R
 
   def self.build
     FileUtils.cd('squashfs-tools') do
       system "sed -i '5iLZ4_SUPPORT = 1' Makefile"
       system "sed -i '6iLZO_SUPPORT = 1' Makefile"
       system "sed -i '7iXZ_SUPPORT = 1' Makefile"
+      system "sed -i '8iZSTD_SUPPORT = 1' Makefile"
       system 'make'
     end
   end
 
   def self.install
```

Comment:
From a quick glance at the repo, it looks like it has support for installing via the makefile (make install). Is there a reason why we don't do that?

Reply:
Good question...

Reply:
Rebuilt this using `make install`...